nomos-core/src/da/mod.rs

```diff
@@ -25,4 +25,6 @@ pub trait DaProtocol {
         &self,
         attestations: S,
     ) -> Self::Certificate;
+
+    fn validate_certificate(certificate: &Self::Certificate) -> bool;
```

Comment:
should this have the blob in input?

Reply:
I'm not really sure, actually. Maybe we need to validate the certificate over the original data?
In this case certificate is the aggregation of attestations that proofs that data is distributed. Either we can check it stand alone or for all blobs (?)

Reply:
You're right, I was confusing those terms. I think the fact that it might be sharded it's a protocol detail (e.g. we do that with rs encoding but not with full replication), while the concept of bl-blobs is independent of the way we do data availability, it's just a piece of data that is available somewhere.

Reply:
But anyway, in the case of a full replication we would just create a `Blob` that contains the same data for each of them (encoding non-encoding 😂 ). I think for now we can assume that the certificate can be validated by itself, and we can change it later if we see fit?